test_scripts/Polices/appID_Management/163_ATF_P_TC_HMI_Status_Value_Of_AppId_In_PT_Is_Null.lua

```diff
@@ -75,8 +75,8 @@ end
 
 function Test:Precondition_UpdatePolicy()
   testCasesForPolicyAppIdManagament:updatePolicyTable(self, "files/jsons/Policies/appID_Management/ptu_013_2.json")
-  EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", { appID =  data.params.application.appID, appRevoked =  true})
-  EXPECT_HMICALL("BasicCommunication.ActivateApp")
+  EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", { appID =  HMIAppID, appRevoked =  true})
+  EXPECT_HMICALL("BasicCommunication.ActivateApp", { hmiLevel = "NONE" })
```

Comment:
@pold500, According to https://github.com/smartdevicelink/sdl_core/blob/feature/extended_policy_functionality/src/components/interfaces/HMI_API.xml name of the parameter is "level"